build/build.js

```diff
@@ -0,0 +1,70 @@
+const babel = require('babel-core');
```

Comment:
Wouldn't this be a good point to introduce `gulp`?

Reply:
Considering the needs we have, I don't think we require a task runners like gulp ATM. I don't see any needs other tasks than very simple transpiling. Gulp would be just a wrapper that replaces read and write functions at the moment. 

Still in case we need something more complex we can definitely re-write this with gulp.